Multisig/UI/WalletConnect/Client/WalletConnectClientController.swift

```diff
@@ -135,27 +135,27 @@ class WalletConnectClientController {
         }
     }
 
-    func sign(transaction: Client.Transaction, completion: @escaping (String?) -> Void) {
+    func send(transaction: Client.Transaction, completion: @escaping (String?) -> Void) {
```

Comment:
I think it will break signing with WC keys. 
Execution is in the `execute` method.

Reply:
this was an additional custom method that I added while working on the release, it will not touch anything else

Reply:
Should we remove 'execute' method?

Reply:
not at the moment, I don't expect this to be a WC refactoring, I think this should be done in a separate task